dataikuapi/dss/notebook.py

```diff
@@ -27,14 +28,20 @@ def unload(self, session_id=None):
                 raise Exception("Several sessions of the notebook are running, choose one")
             else:
                 session_id = state['activeSessions'][0].get('sessionId', None)
-        return self.client._perform_json("DELETE", "/projects/%s/notebooks/" % self.project_key, params={'notebookName' : self.notebook_name, 'sessionId' : session_id})
+        return self.client._perform_json("DELETE",
+                                         "/projects/%s/jupyter-notebooks/%s/sessions/%s" % (self.project_key, self.notebook_name, session_id))
 
     def get_state(self):
         """
         Get the status of the notebook
         """
-        if self.state is None:
-            self.state = self.client._perform_json("GET", "/projects/%s/notebooks/" % self.project_key, params={'notebookName' : self.notebook_name})
+        notebook_list = self.client._perform_json("GET",
```

Comment:
This variable contains the metadata of the notebook, not a notebook.
The java object is names `JupyterNotebookListItem` which is super confusing so maybe let's not use that.
Maybe something like:
```suggestion
        notebook_states = self.client._perform_json("GET",
```